src/main/java/org/candlepin/resource/UserResource.java

```diff
@@ -196,6 +196,17 @@ public Stream<OwnerDTO> listUserOwners(@Verify(User.class) String username) {
         return null;
     }
 
+
+    /**
+     * Retrieve the primary owner the user can register systems to.
+     */
+    @Override
+    public OwnerDTO getPrimaryOwner(@Verify(User.class) String username) {
```

Comment:
Instead of adding a new API on our side, I was thinking we just roll this change into the existing `listUserOwners` endpoint. It will still be returning a Stream<OwnerDTO>, but we'll always be returning a single owner in that list (the primary owner from `user.getPrimaryOwner()`. 

Satellite supports multiple owners, but they don't actually use this API (they have their own Katello API that handles the fetching of the list of owners), so we wouldn't be breaking them. On the other hand, this API is being used in Hosted, and from previous discussions I remember they already have a way to discern a 'primary' owner (they just didn't have a way to communicate this to us, which is what this PR addresses).